src/modules/launcher/Plugins/Microsoft.Plugin.Folder/plugin.json

```diff
@@ -9,5 +9,6 @@
   "Website": "https://aka.ms/powertoys",
   "ExecuteFileName": "Microsoft.Plugin.Folder.dll",
   "IcoPathDark": "Images\\folder.dark.png",
-  "IcoPathLight": "Images\\folder.light.png"
+  "IcoPathLight": "Images\\folder.light.png",
```

Comment:
Are these still needed?

Reply:
Yeah, in Run itself we use images instead of glyphs.

Reply:
Is it a technical reason to not use glyphs there as well?

Reply:
Yes. We only have one ItemTemplate that we use for Run, that requires an image.

It would be nice to use a ItemTemplateSelector and have 2 (or more) ItemTemplates. Gives us more freedom to change how content is displayed per plug-in.

Reply:
> Yes. We only have one ItemTemplate that we use for Run, that requires an image.
> 
> It would be nice to use a ItemTemplateSelector and have 2 (or more) ItemTemplates. Gives us more freedom to change how content is displayed per plug-in.

This would resolve the problem where we don't have/find a good glyph for a plugin.